### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by kjellmf** [kjellmf/dot2tex#8]
Dot2tex is a relatively old project and it was also one of my first Python projects. The quality of the code base is therefore not the best :) The code therefore needs to be cleaned up a bit before I will start porting it to Python 3. My plan is to support both 2.7 and 3.x from the same code base.

**Comment by DavidNemeskey** [comment]
If it doesn't work for Python 3, you should make sure people cannot install it from PyPI to a Python 3 environment. I don't know how to do it, maybe specify
```
Programming Language :: Python :: 2.6
Programming Language :: Python :: 2.7
```
in `setup.py`?

**Comment by tscrim** [comment]
See #68.

**Comment by kjellmf** [comment]
Fixed in #68
